### PR TITLE
Splits crash categories based on importance

### LINKED ISSUE #287: Instead of displaying the various crash categories alphabatically.....
Let's display crash categories based on importance level, E.g. list them as:
- main_crashes
- content_crashes
- startup_crashes

### PR DESCRIPTION
**Pull request by kagehina919**
Fixes #287.
Split measures into crash categories and others in `subview.jsx` based on the ORDER constant defined in `schema.js`. Also fixed alphabetical sorting for other measures.
screenshot from 2018-11-01 16-47-09
@wlach please review.

### CODE REVIEW

File: frontend/ui/subview.jsx [hunk 1]
```diff
@@ -93,6 +96,23 @@ export class SubViewComponent extends React.Component {
...
   }
 
+  splitByOrder(measures) {
+    const main = [];
+
+    for (let i = 0; i < ORDER.length; i += 1) {
+      main.push(measures.find(element => element.name === ORDER[i]));
+    }
+
+    const others = _.orderBy(_.difference(measures, main), ['name']);
```
Comment (wlach) [inline frontend/ui/subview.jsx]: others -> otherMeasures

File: frontend/ui/subview.jsx [hunk 2]
```diff
@@ -170,42 +191,88 @@ export class SubViewComponent extends React.Component {
...
+                      <td al>
+                        {measure.lastUpdated
+                          ? moment(measure.lastUpdated).fromNow()
+                          : 'N/A'}
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+              <h3>Others</h3>
```
Comment (wlach) [inline frontend/ui/subview.jsx]: Others -> Other Measures

File: frontend/ui/subview.jsx [hunk 3]
```diff
@@ -170,42 +191,88 @@ export class SubViewComponent extends React.Component {
...
-                              this.state.countType,
-                              this.state.timeWindow
-                            )}
-                          </td>
-                        ))}
-                        <td al>
-                          {measure.lastUpdated
-                            ? moment(measure.lastUpdated).fromNow()
-                            : 'N/A'}
+                  {this.state.measures.main.map(measure => (
```
Comment (wlach) [inline frontend/ui/subview.jsx]: We should probably factor this into a react component like `MeasureTable` which takes a `measures` argument as input, to reduce duplication and make it easier to navigate the code. See the `PlatformCard.jsx` file for an example of this sort of thing.

File: frontend/schema.js [hunk 4]
```diff
@@ -54,6 +54,15 @@ export const MAX_VISIBLE_SERIES = {
 };
 
 export const KEY_MEASURES = ['content_crashes', 'main_crashes'];
+export const ORDER = [
```
Comment (wlach) [inline frontend/schema.js]: Let's give this variable a more descriptive name like `CRASH_MEASURE_ORDER`

File: frontend/ui/subview.jsx (hunk 3 above)
Comment (kagehina919) [reply frontend/ui/subview.jsx]: Updated. :+1:

**Pull request merged**